containers/java/getLineCoverageFor.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I think we should rename this script to better describe intent: `fetch_repo_and_collect_coverage.sh`
For scripts I prefer to us underscores rather than camelcase.